src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -143,6 +143,13 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Analyzers.NGram.MaxGram", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public int MaxGram { get; set; }
 
+        [Description("Managed allocations limit, which an index will stop and complete the current batch")]
+        [DefaultValue(512)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [IndexUpdateType(IndexUpdateType.Refresh)]
+        [ConfigurationEntry("Indexing.MangedAllocationsBatchLimitInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public Size? MangedAllocationsBatchLimit { get; protected set; }
```

Comment:
How do I set it to null?

Reply:
I think one way is from the `settings.json` to do:
```
"Indexing.MangedAllocationsBatchLimitInMb":null
```

Reply:
Does it work like taht or it will switch to 512?